src/plugins/mutualGroupDMs/index.tsx

```diff
@@ -76,6 +76,17 @@ export default definePlugin({
 
     isBotOrSelf: (user: User) => user.bot || user.id === UserStore.getCurrentUser().id,
 
+    getMutualGDMCountText: (user: User) => {
+        const count = ChannelStore.getSortedPrivateChannels().filter(c => c.isGroupDM() && c.recipients.includes(user.id)).length;
+        if (count === 0) {
+            return "No Mutual Groups";
+        } else if (count === 1) {
+            return "1 Mutual Group";
+        } else {
+            return `${count} Mutual Groups`;
+        }
```

Comment:
Why not just put "0 Mutual Groups"?
```suggestion
return `${count} Mutual Group ${count!==1?"s":""}`
```
idk how other people would feel about this minification, but :shrug:

Reply:
I don't remember how it is but it should match how mutual friends and servers do it